main.py

```diff
@@ -6,7 +6,7 @@
 from scraper import scrape
 from roadrunner import roadrunner
 from webstemmer import webstemmer
-from scrapy import scrapy
+from scrapy_runner import scrapy
```

Comment:
Tist drugi error, ki sva ga imela, ko je bil PYTHONPATH pravilno nastavljen je bil pa zaradi tega. `scrapy` je ime paketa k ga instaliras s pipom, ampak ker je tud ime tvojga modula je prslo do collisiona. `scrapy` command je zdj executal tvoj scrapp.py namest scrappy module v virtualnem folderju.

Fix je dokaj simple. Tvoj scrapy.py sem preimenoval v scrapy_runner, tko da ni vec collisiona in zdj pa zadeva dela!